src/plugin/index-more/src/java/org/apache/nutch/indexer/more/MoreIndexingFilter.java

```diff
@@ -316,6 +324,39 @@ public void setConf(Configuration conf) {
         LOG.error(org.apache.hadoop.util.StringUtils.stringifyException(e));
       }
     }
+    
+    URL res = conf.getResource("date-styles.txt");
```

Comment:
Maybe a more specific file name?  So, that the usage of the resource is clear to every (new) user - since the `conf/` folder is shared for all Nutch components and plugins. Additionally, a comment in the config file might be useful which explains what it is used for.

Reply:
Just in case: I meant the file name, not the variable name. A typical Nutch user only edits the configuration file but does not look into the source code to figure out what the file is used for.

Reply:
ok, any suggestion? the only default file naming thing I can see is that every one else is creating a template instead really/directly uses a file.
so what about:
index-more-last-modified-date-styles.txt
?

Reply:
Well, that sounds too long. Maybe only index-more-date-styles.txt? But we can leave it as the usage is now described in the file itself.

Yes, you're right about the *.template : if someone builds Nutch from the source code all config files in `conf/`are "instantiated" during the first compile from the *.template versions. That's to avoid that the version control system complains about modified configuration files if users need to adapt the settings for their specific use case. Could you rename the file to *.template? - it's then automatically copied when `ant runtime` is called.